packages/mattermost-redux/src/actions/apps.ts

```diff
@@ -12,8 +14,13 @@ import {bindClientFunc} from './helpers';
 // susceptible to breaking changes without pushing the major version of this package.
 
 export function fetchAppBindings(userID: string, channelID: string): ActionFunc {
-    return bindClientFunc({
-        clientFunc: () => Client4.getAppsBindings(userID, channelID),
-        onSuccess: AppsTypes.RECEIVED_APP_BINDINGS,
-    });
+    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
+        const channel = getChannel(getState(), channelID);
+        const teamID = channel?.team_id || '';
```

Comment:
Does it make sense / is it desirable to send the current team id if the channel does not have a team id?